### DISCUSSION TITLE
How to organize our workforce?

### DISCUSSION THREAD

**Discussion by Flyingmana** [OpenMage/magento-lts#2908] [General]
> the team should be on a "turn" basis, it's not right that there are these many dormant roles in the team, maintainers that are not longer involved should step down, every role and privilege should have an expiration date, elections, meetings

As a start for this request I created an overview of all the roles Iam aware of. This is for sure not complete, so I would be thankful for a bit of support here.
https://github.com/OpenMage/organizational/blob/main/Roles_and_Repsonsibilities.md

There is also the question how we want to assign responsibility, and more importantly, make sure we are all moving towards the same Goal. So I guess we also need to define and write down Goals, so we can actually identify wrong decisions.

What approaches to this are you all suggesting?

**Comment by mark-netalico** [comment]
I'm fine with continuing to take point on security matters.

**Comment by Flyingmana** [comment]
do I see it correct, that most of our volunteers are not really interested to take over specific roles?

**Reply by fballiano** [reply]
I'm sorry for not answering, I think I missed this post somehow, I'd feel fit for the last 3 positions in the list but at the moment we miss a lot of people for the rest of the positions :-\

**Reply by Tomasz-Silpion** [reply]
Yep, due lack of updated this discussion was hidden for me as well. As I've recently purchased openmage.dev domain for my own needs and started to host sample OpenMage store there I can fit for Demo Store Maintainer. With 15 years in industry starting from Magento 1.4 and people from Czech Republic and Ukraine I employ I can take care of hosting partner management / public relations for Middle East Europe and as well test team if needed.

**Reply by Flyingmana** [reply]
> Yep, due lack of updated this discussion was hidden for me as well. As I've recently purchased openmage.dev domain for my own needs and started to host sample OpenMage store there I can fit for Demo Store Maintainer. With 15 years in industry starting from Magento 1.4 and people from Czech Republic and Ukraine I employ I can take care of hosting partner management / public relations for Middle East Europe and as well test team if needed.

this would be a great help, we should discuss some details. What is the best communication channel for you?
Generally we use Discord heavily for fast communication and coordination.

**Reply by Tomasz-Silpion** [reply]
Thank you for the trust and for adding me to the organization members and responsibilities document. I've joined Discord and the OpenMage channel today with 'silpionpl' as my nickname. It might be challenging for me to meet all the expectations, so I've asked my former employee [@Domirak](https://github.com/xrx415)  to assist with the partner management/public relations role, as he's a bit more representative than I am, considering I've spent half of my life engrossed in M1/M2 coding in the proverbial basement. His Discord nickname is xrx415. And once again, as we are both located in same area in Poland, close to the Czechia border and some relatives there and in Ukraine, remember that PR/PM roles, at least for other regions still need to be filled. We hope to host "Meet OpenMage" event in the coming years, so long live the community.

Meanwhile, I will ensure that the [openmage.dev](https://openmage.dev) and [openmage.store ](https://openmage.store) domains are always available and up to date.

**Comment by mark-netalico** [comment]
I'm still fine to be the Security POC.

**Comment by elidrissidev** [comment]
I had been transitioning jobs recently so I was very busy, but I will have more free time now. I can take a part of "Code Maintainer - OpenMage" and "Website Maintainer" roles.

**Comment by kiatng** [comment]
I am a part of the Code Maintainers, and will continue to review components that I am familiar with.

**Comment by addison74** [comment]
As far as I am concerned, even if I had a very difficult year from all points of view and which required me enormously, I am with OpenMage and I can cover the areas where I have knowledge. The most important aspect would be to test all PRs and confirm or not the reported issues. I know it's low-level work and requires effort, but we will have to bring many things up to date. Each of us can contribute something and I think that OpenMage is still an alternative to be considered to the shopping carts that have become alternatives for those who have not embraced the Magento 2 platform.
